src/client/clientlauncher.cpp

```diff
@@ -444,6 +471,13 @@ bool ClientLauncher::launch_game(std::string &error_message,
 			server["port"] = menudata.port;
 			server["description"] = menudata.serverdescription;
 			ServerList::insert(server);
+
+	#if USE_DISCORD
+			g_discord->setDetails("Playing on " + menudata.address + ":" +
+					      menudata.port + " server");
+			g_discord->setState(menudata.serverdescription);
```

Comment:
Should be "Playing solo" or "Playing online"

https://discordapp.com/developers/docs/rich-presence/how-to#updating-presence-update-presence-payload-fields

Reply:
same